build.gradle

```diff
@@ -80,7 +80,7 @@ task javadocJar(type: Jar) {
     from javadoc.destinationDir
 }
 
-version '1.0.0.0'
+version '1.0.0-alpha1'
```

Comment:
Lets leave this as plugin version `1.0.0.0`. As this will the version which we will use for beta1.